src/frontend/packages/lib_video/src/api/useTimedTextMetadata/index.ts

```diff
@@ -0,0 +1,37 @@
+import { fetchOne, metadata, TimedText } from 'lib-components';
+import { useQuery, UseQueryOptions } from 'react-query';
+
+import { TimedTextMetadata } from 'types/metadata';
+
+export const useTimedText = (
+  id: string,
+  queryConfig?: UseQueryOptions<TimedText, 'timedtexttracks', TimedText>,
+) => {
+  const key = ['timedtexttracks', id];
+  return useQuery<TimedText, 'timedtexttracks'>(key, fetchOne, queryConfig);
+};
```

Comment:
Should not be in this module I think.

Reply:
Where does it have to be then ? The api folder seemed quite relevant for me